tests/unit/core/test_installAddon.js

```diff
@@ -601,6 +601,42 @@ describe(__filename, () => {
         .returns(false);
     });
 
+    describe('isAddonEnabled', () => {
+      it('returns true when the add-on is enabled', async () => {
+        const fakeAddonManager = getFakeAddonManagerWrapper();
+        const addon = createInternalAddon(fakeAddon);
+
+        const { root } = renderWithInstallHelpers({
+          addon,
+          _addonManager: fakeAddonManager,
+        });
+        const { isAddonEnabled } = root.instance().props;
+        const isEnabled = await isAddonEnabled();
+
+        sinon.assert.calledWith(fakeAddonManager.getAddon, addon.guid);
+        expect(isEnabled).toEqual(true);
+      });
+
+      it('returns false when there is an error', async () => {
+        const fakeAddonManager = getFakeAddonManagerWrapper({
+          // Resolve a null addon which will trigger an exception.
+          getAddon: Promise.resolve(null),
+        });
+
+        const { root } = renderWithInstallHelpers({
+          addon: createInternalAddon({
+            addon: null,
```

Comment:
this does not work, it should be null directly and not createInternalAddon.

Reply:
I did initially try this but I see this without doing it this way:

```bash
 TypeError: Cannot read property 'guid' of null

      337 |     const {
      338 |       _addonManager,
    > 339 |       addon: { guid },
          |                ^
      340 |     } = this.props;
```

Reply:
should I be handling this another way in installAddon?

Reply:
I see...I'll update..thanks